version/version.go

```diff
@@ -1,4 +1,58 @@
 package version
 
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/google/renameio"
+)
+
 // Version is the version of the build.
-const Version = "1.14.0-dev"
+const Version = "1.14.2-dev"
```

Comment:
We cut 1.14.2, so might want to change this.

Reply:
hm, do we even update the version file in master as we cut branches? I think this change isn't supposed to be here...

Reply:
My bad actually, we cut in https://github.com/cri-o/cri-o/pull/2435 but it hasn't gone in yet. And yeah, master keeps track of the `-dev` versions, so it should be there.

Reply:
so we can resolve this?